src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -259,6 +252,43 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         });
     }
 
+    /**
+     * @notice Updates the stakes of one or more operators in the StakeRegistry, for each quorum
+     * the operator is registered for.
+     * 
+     * If any operator no longer meets the minimum stake required to remain in the quorum,
+     * they are deregistered.
+     */
+    function updateOperators(address[] calldata operators) external onlyWhenNotPaused(PAUSED_UPDATE_OPERATOR) {
+        for (uint256 i = 0; i < operators.length; i++) {
+
+            address operator = operators[i];
+            OperatorInfo storage operatorInfo = _operatorInfo[operator];
+            bytes32 operatorId = operatorInfo.operatorId;
+
+            // Only update operators currently registered for at least one quorum
+            if (operatorInfo.status != OperatorStatus.REGISTERED) {
+                continue;
+            }
+
+            uint192 currentBitmap = _currentOperatorBitmap(operatorId);
+            bytes memory currentQuorums = BitmapUtils.bitmapToBytesArray(currentBitmap);
+
+            /**
+             * Update the operator's stake for their active quorums. The stakeRegistry returns a bitmap
+             * of quorums where the operator no longer meets the minimum stake, and should be deregistered.
+             */
+            (uint192 quorumsToRemove) = stakeRegistry.updateOperatorStake(operator, operatorId, currentQuorums);
```

Comment:
why parentheses

Reply:
from an earlier change i think? anyway, ill remove em.

Reply:
Done in [23eb1a3](https://github.com/Layr-Labs/eigenlayer-middleware/pull/57/commits/23eb1a3e7cf4f2e4f81af138d307a31355e56cb5)